prijateli_tree/app/routers/games.py

```diff
@@ -400,18 +416,37 @@ def route_add_score(
     """
     Function that updates the player's score in the database
     """
+    game, player = get_game_and_player(game_id, player_id, db)
+    game_status = did_player_win(game, player_id, db)
+    session_player = get_session_player_from_player(player, db)
+
+    session_player.correct_answers += game_status["is_correct"]
+    session_player.points += game_status["is_correct"] * WINNING_SCORE
+    db.commit()
+    db.refresh(session_player)
```

Comment:
To make sure this can't be done twice, I'd add this logic to the game finishing endpoint. Making it a discrete endpoint makes this super easy to accidentally increment too much.

Reply:
I'd like to hear more about your thoughts on this. Let's talk about this at the meeting tomorrow.  Ad for me, I think we still have the potential for problems of double tapping if it's in the end of game endpoint and I like having post/put and get maintain distinct purposes.